modules/apple_support/0.11.0/patches/change_to_relative_labels.patch

```diff
@@ -0,0 +1,58 @@
+From 4fa95444ff3681667c42942dab24e6b9f71ab32a Mon Sep 17 00:00:00 2001
+From: Patrick Balestra <me@patrickbalestra.com>
+Date: Thu, 20 Jan 2022 12:06:13 +0100
+Subject: [PATCH] Change to relative labels
+
+---
+ lib/apple_support.bzl       | 2 +-
+ test/apple_support_test.bzl | 2 +-
+ test/xcode_support_test.bzl | 4 ++--
+ 3 files changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/lib/apple_support.bzl b/lib/apple_support.bzl
+index bd7c50b..33c9cbb 100644
+--- a/lib/apple_support.bzl
++++ b/lib/apple_support.bzl
+@@ -160,7 +160,7 @@
+         "_xcode_path_wrapper": attr.label(
+             cfg = "host",
+             executable = True,
+-            default = Label("@build_bazel_apple_support//tools:xcode_path_wrapper"),
++            default = "//tools:xcode_path_wrapper",
+         ),
+     }
+ 
+diff --git a/test/apple_support_test.bzl b/test/apple_support_test.bzl
+index 9efc5cc..aabb373 100644
+--- a/test/apple_support_test.bzl
++++ b/test/apple_support_test.bzl
+@@ -14,7 +14,7 @@
+ """Definition of a test rule to test apple_support."""
+ 
+ load(
+-    "@build_bazel_apple_support//lib:apple_support.bzl",
```

Comment:
The whole patch file will be upstreamed, right?

Reply:
Yes I will!